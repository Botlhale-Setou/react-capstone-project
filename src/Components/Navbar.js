import React from "react";
import logo from "../assets/logo.png";

const Navbar = () => {
  return (
    <div className="navbar">
      <img src={logo} alt='Mapfrica logo' className="logo" />
			<h1 className="siteTitle">MAPFRICA</h1>
    </div>
  );
};

export default Navbar;
