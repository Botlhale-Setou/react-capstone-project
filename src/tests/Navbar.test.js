import React from 'react';
import Renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import '@testing-library/jest-dom';
import Navbar from '../Components/Navbar';

describe('Test Navbar', () => {
  test('Navbar render', () => {
    const tree = Renderer
      .create(
        <Router>
          <Navbar />
        </Router>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
