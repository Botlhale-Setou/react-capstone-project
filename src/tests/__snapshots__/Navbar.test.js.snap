// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Navbar Navbar render 1`] = `
<div
  className="navbar"
>
  <img
    alt="Mapfrica logo"
    className="logo"
    src="logo.png"
  />
  <h1
    className="siteTitle"
  >
    MAPFRICA
  </h1>
</div>
`;
